roles/sigstore_scaffolding/tasks/podman/tuf.yml

```diff
@@ -23,8 +23,8 @@
     validate_certs: false
     return_content: true
   until: "rekor_public_key_result.status == 200"
-  retries: 5
```

Comment:
Do we want to make these and the others configurable as different operating environment will have different failure rates (time to pull down the image etc)